src/routes.json

```diff
@@ -148,18 +148,16 @@
         "title": "Ideas"
     },
     {
-        "name": "jobs",
+        "name": "jobs-redirect",
         "pattern": "^/jobs/?$",
         "routeAlias": "/jobs(/moderator)?/?$",
-        "view": "jobs/jobs",
-        "title": "Jobs"
+        "redirect": "https://www.scratchfoundation.org/opportunities/"
     },
     {
-        "name": "jobs-moderator",
+        "name": "jobs-redirect2",
```

Comment:
You can remove this whole block, since we were only linking to it internally and now will only be linking to `/jobs`

Reply:
How do you know that? Someone would have linked that page and it's not good to make it 404.

Reply:
Ah! Makes sense, you mean merging it to jobs?

Reply:
✔️ Done